test_scripts/RC/commonRC.lua

```diff
@@ -44,8 +45,7 @@ function commonRC.getRCAppConfig()
     priority = "NONE",
     default_hmi = "NONE",
     moduleType = { "RADIO", "CLIMATE" },
-    groups = { "Base-4" },
-    groups_primaryRC = { "Base-4", "RemoteControl" },
+    groups = { "Base-4", "RemoteControl" },
```

Comment:
@aderiabin General comment for all scripts in this PR:
Release of the module means not only possibility to set the data by another app
But also impossibility getting of OnInteriorVehicleData notifications for this module if we do this before 
Suggestion is to expand the tests on such verification

Reply:
@dboltovskyi  Verification was added in ed49913